src/plugins/intel_gna/src/backend/am_intel_dnn.cpp

```diff
@@ -46,20 +45,18 @@
   */
 #define LIGHT_DUMP
 
-using namespace GNAPluginNS::backend;
-using namespace ov::intel_gna;
+using gna_convolution_layer::outputFromConv;
+using gna_convolution_layer::outputFromPooling;
```

Comment:
consider to move to GNA plugin namspace (ov::intel_gna)

Reply:
Will be fixed in next PR.